packages/backend/src/auth/jwtStrategy.ts

```diff
@@ -16,11 +16,7 @@ export const jwtStrategy = new JWTStrategy(
   },
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   async (req: Request, jwtPayload: any, done: any) => {
-    const { exp, userId, email, noKyc } = jwtPayload
-
-    if (Date.now() > exp * 1000) {
```

Comment:
I am expecting passport's jwtStrategy to already throw and not reach here in case the token is expired.
(since the expiration is also encoded in the token itself).
As such, this check should be redundant